webapp/components/LoginForm/LoginForm.spec.js

```diff
@@ -57,5 +58,43 @@ describe('LoginForm', () => {
         })
       })
     })
+
+    describe('Click on show password input field type change', () => {
+      const wrapper = Wrapper()
+      it('does not show the password by default', () => {
+        expect(wrapper.find('input[name ="password"]').attributes('type')).toEqual('password')
+      })
+
+      it('shows the password after click on show password', async () => {
+        wrapper.find('span.click-wrapper').trigger('click')
+        await Vue.nextTick()
+        await expect(wrapper.find('input[name = "password"]').attributes('type')).toEqual('text')
+      })
+    })
+
+    describe('Click on show password icon, icon change', () => {
+      const wrapper = Wrapper()
+      it('shows eye icon by default', () => {
+        expect(wrapper.find('span.icon-wrapper').attributes('data-test')).toEqual('eye')
+      })
+
+      it('shows the slash-eye icon after click', async () => {
+        wrapper.find('span.click-wrapper').trigger('click')
+        await Vue.nextTick()
+        await expect(wrapper.find('span.icon-wrapper').attributes('data-test')).toEqual('eye-slash')
+      })
+    })
+
+    describe('Focus returns to password input container after show password click', () => {
+      const wrapper = Wrapper()
+      const componentToGetFocus = wrapper.find('input[name ="password"]')
+      it('Focus is on the password field container after click', async () => {
```

Comment:
it focuses on the ...

Reply:
I adjusted the recommendations but am having an issue pulling up the storybook. It is throwing an error that I am not sure where to start with. 

__child-HtmlWebpackPlugin_0  6.5 KiB  HtmlWebpackPlugin_0  HtmlWebpackPlugin_0

I will look into it more. 

As for the button, I am still having an issue getting the far left of the area clickable.

Reply:
OK. The storybook looks fine. I have no problem in running it on my machine.